contracts/test/StakeTable.t.sol

```diff
@@ -745,4 +755,193 @@ contract StakeTable_register_Test is Test {
         stakeTable.withdrawFunds();
         vm.stopPrank();
     }
+
+    // TESTS FOR CURRENT EPOCH
+    function test_initialEpoch_isZero() public {
+        // assert the current block height is initialBlockHeight
+        uint64 initialBlockHeight = 0;
+        (, uint64 currentBlockHeight,) = lcMock.finalizedState();
+        assertEq(currentBlockHeight, initialBlockHeight);
+
+        // Calculate the expected epoch
+        uint64 expectedEpoch = initialBlockHeight / hotShotBlocksPerEpoch;
```

Comment:
I think the test would be easier to read if it just hardcoded zeroes, wherever zero is expected.

Reply:
makes sense, updated here https://github.com/EspressoSystems/espresso-sequencer/commit/af8841c5fb0b0985a495b8fd82e103775786f6ae

Reply:
I meant more generally l if we use asserts in a test it's usually implied that we are asserting that some value is equal to an expected value. So instead of `assert(epoch, expectedEpoch)` we can write `assert(epoch, 0)` and end up with fewer indirections.

However, if the left and right hand size are some complicated expressions then defining some extra variables is helpful for reading.